ocaml/xenopsd/xc/xenops_server_xen.ml

```diff
@@ -2523,6 +2525,7 @@ module VM = struct
                     )
                 then
                   raise (Xenopsd_error Failed_to_acknowledge_suspend_request) ;
+                with_tracing ~task ~name:"VM_migrate_downtime_begin" Fun.id ;
```

Comment:
You'll see this for suspending as well. You could detect the migration case by matching `data` with `Some (FD _)`.